### ISSUE TITLE
Script to generate synthesized image

### ISSUE THREAD

**Issue by jij7401** [naoto0804/SynShadow#7]
Hello, Thank for sharing your code.  I want to know if there is a script to generate Synthesized image(with shadow-free image, matte, attenuation parameter)!

**Comment by naoto0804** [comment]
Hi, thank you for visiting our repo!
I guess the easiest way is to run only [a part of train.py](https://github.com/naoto0804/SynShadow/blob/main/src/train.py#L1-L16) like `python3 train.py --dataset_mode synth`, and then manually loop over the dataloader, which will return dictionaries containing shadow/shadow-free/matte images. Let me know if you have any problem.

**Comment by jij7401** [comment]
Thank you for your reply!
after running a part of train.py like python3 train.py --dataset_mode synth, I can show the image for returning dictionaries containing  shadow/shadow-free/matte images? Or I have to mapping the dictionaries?

**Comment by naoto0804** [comment]
Although I haven't tested yet, according to [here](https://github.com/naoto0804/SynShadow/blob/main/src/data/synth_dataset.py#L67-L70), the dictionary's key should be ['input' (=shadow-free), 'mask'(=matte), 'target'(=shadow)], and values are 3D torch tensor containing image values ranging from 0.0 to 1.0 in (C, H, W) format
